sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -82,54 +79,40 @@ public void define(Context context) {
 
         PropertyDefinition.builder(PHPUNIT_COVERAGE_REPORT_PATH_KEY)
           .name("Coverage Report")
-          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "code coverage"))
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(), "code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build(),
 
         PropertyDefinition.builder(PHPUNIT_IT_COVERAGE_REPORT_PATH_KEY)
           .name("IT Coverage Report")
-          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "integration test code coverage"))
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(), "integration test code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build(),
 
         PropertyDefinition.builder(PHPUNIT_OVERALL_COVERAGE_REPORT_PATH_KEY)
           .name("Overall Coverage Report")
-          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(context), "overall code coverage"))
+          .description(String.format(REPORT_PATH_DESCRIPTION_TEMPLATE, deprecationMessage(), "overall code coverage"))
           .onQualifiers(Qualifiers.MODULE, Qualifiers.PROJECT)
           .category(PHP_CATEGORY)
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build());
     }
-    if (versionSupportsMultiPathCoverageReport(context)) {
-      context.addExtension(PropertyDefinition.builder(PHPUNIT_COVERAGE_REPORT_PATHS_KEY)
```

Comment:
Sounds like a mistake on my side, because the method : 
```
  private static boolean versionSupportsMultiPathCoverageReport(Context context) {
    return context.getSonarQubeVersion().isGreaterThanOrEqual(SQ_VERSION_6_2);
  }

``` 

always returns true.

I am genuinely suprised to not have seen a failing IT.